src/library/dlgtagfetcher.cpp

```diff
@@ -47,13 +73,27 @@ QStringList trackReleaseColumnValues(
 
 void addTrack(
         const QStringList& trackRow,
-        int resultIndex,
+        int tagIndex,
         QTreeWidget* parent) {
     QTreeWidgetItem* item = new QTreeWidgetItem(parent, trackRow);
-    item->setData(0, Qt::UserRole, resultIndex);
+    item->setData(0, Qt::UserRole, tagIndex);
     item->setData(0, Qt::TextAlignmentRole, Qt::AlignLeft);
 }
 
+void updateOriginalTag(const Track& track, QTreeWidget* parent) {
```

Comment:
prefix pointer with p

Reply:
Then it should be changed in other functions too, I will rename all.